LICENSE.md

```diff
@@ -0,0 +1,21 @@
+## Public Domain
+
+This project constitutes a work of the United States Government and is not subject to domestic copyright protection under 17 USC § 105. Additionally, we waive copyright and related rights in the work worldwide through the [CC0 1.0 Universal public domain dedication](https://creativecommons.org/publicdomain/zero/1.0/).
+
+All contributions to this project will be released under the CC0 dedication. By submitting a pull request, you are agreeing to comply with this waiver of copyright interest. See [CONTRIBUTING](CONTRIBUTING.md) for more information. 
+
+## GNU General Public License
```

Comment:
This section should be removed, since the project is available under CC0/Public Domain. If we're pulling in GPL code, we should discuss first.

Reply:
I [deleted this lines and the GPL part](https://github.com/datopian/ckanext-datajson/commit/2eb967d92220b9fb8921694bd2a5bf0bb09e8443).
Let me know if it's ok